src/theory/arith/theory_arith_private.cpp

```diff
@@ -4782,8 +4782,11 @@ std::pair<bool, Node> TheoryArithPrivate::entailmentCheck(TNode lit, const Arith
   return make_pair(false, Node::null());
 }
 
-bool TheoryArithPrivate::decomposeTerm(Node term, Rational& m, Node& p, Rational& c){
-  Node t = Rewriter::rewrite(term);
+bool TheoryArithPrivate::decomposeTerm(Node t,
```

Comment:
I assume the rewrite was removed here since it always is already rewritten? Maybe change to assertion?

Reply:
This is true. For an assertion like `Assert(t == rewrite(t))` we'd still need access to the rewriter, but `decomposeTerm` is still static. I didn't want to make it non-static just to add an assertion...

Reply:
Ah I didnt realize it was static.